pages/cookbook/primitive-types.mdx

```diff
@@ -0,0 +1,243 @@
+# Primitive types
```

Comment:
the primitive types section is not really a collection of recipes, so its contents needs to be eventually moved to the Book sections

Reply:
Agreed